charts/unified-chart/values.yaml

```diff
@@ -0,0 +1,110 @@
+# Default values for frontegg unified chart.
+
+# required
+# serviceName: my-new-awesome-service
+# team: my-team-of-imbeciles
+
+componentsCollectionIdentifier: local
+
+minReplicasOverride:
```

Comment:
we won't need this, but i guess you keep it for legacy issues?
But we can use the new way from the start